README.md

```diff
@@ -431,7 +525,14 @@ Future.try(() => data.foo.bar.baz)
 This function has an alias `attempt`, for environments in which `try` is a reserved word.
 
 #### tryP
-##### `.tryP :: (() -> Promise e r) -> Future e r`
+
+<details><summary><code>tryP :: (() -> Promise e r) -> Future e r</code></summary>
+
+```hs
+tryP :: (() -> Promise e r) -> Future e r
```

Comment:
Actually I can just replace instances of `-> ()` by `-> x` and I won't need a special `Undefined` type. The return value of these functions can be anything, but is ignored.